ci/makefiles/android.mk

```diff
@@ -0,0 +1,78 @@
+# Downloads and installs the Android SDK depending on supplied platform: darwin or linux
+
+# We must provide a platform (darwin or linux) and we need JAVA_HOME defined
+ifndef target_os
+    $(error target_os is not set...aborted!)
+endif
+
+# Those android NDK/SDK variables can be override when running the file
+ifndef ANDROID_NDK_VERSION
+    ANDROID_NDK_VERSION=17c
+endif
```

Comment:
What about the shorthand default? e.g.
```
ANDROID_NDK_VERSION ?= 17c
```

Reply:
Also changed other entries from `android.mk` and `osx.mk` files :wink:

@AndreMiras, thank you for the tip!!!  :smile: